### PR TITLE
ANNOUNCEMENT: Upgrade Your Nodes for the Spiral Upgrade on Block 19,250,000

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is a post to announce that node operators and miners must upgrade their nodes for the SPIRAL upgrade.

"ANNOUNCEMENT: Upgrade Your Nodes for the Ethereum Classic Spiral Hard Fork on Block 19,250,000"

EN.

CN coming soon.

This announcement will be published on December 21 of 2023, so we have a 14 day lead.

EDIT: This announcement will be published on January 16 of 2024.

### CODE REVIEW

**Review by Plentywolf — APPROVED** [review]

**Review by brolal — DISMISSED** [review]

**Comment by DonaldMcIntyre** [comment]
> CC: @brolal @gitr0n1n 
> 
> 
> 
> As we already have a [blog post](https://ethereumclassic.org/blog/2023-12-09-spiral-fork-announcement) for this initial announcement, perhaps we should hold off on this for now and post it in early January to get everyone's attention again while they are in work mode after the holidays?

I agree, I will reschedule it for mid January.

**Review by gitr0n1n — DISMISSED** [review]
Approved, don't merge until January after the holidays when everyone is at their computers again please.

**Review by gitr0n1n — APPROVED** [review]

**Review by brolal — APPROVED** [review]

**Review by Plentywolf — APPROVED** [review]

**Review by Plentywolf — APPROVED** [review]
Ayo

**Pull request merged**
